data/World/Overworld.json

```diff
@@ -1714,6 +1950,10 @@
                 is_child or can_use(Hover_Boots) or (is_adult and logic_zora_river_lower)",
             "ZR Near Domain Freestanding PoH": "
                 is_child or can_use(Hover_Boots) or (is_adult and logic_zora_river_upper)",
+            "ZR Waterfall Red Rupee 1": "is_adult",
+            "ZR Waterfall Red Rupee 2": "is_adult",
+            "ZR Waterfall Red Rupee 3": "is_adult",
+            "ZR Waterfall Red Rupee 4": "is_adult",
```

Comment:
I'd suggest for consistency, these probably have to require Iron Boots if a new jump dive trick isn't on. You might want to ask around what people think of the current jump dive tricks, like for example the one in Jabu. I would say the one in Jabu is possibly easier than collecting all of these rupees?

Reply:
These are particularly easy to get, and fairly intuitive unlike something like the Jabu jumps. The jump is not precise at all. From the top on the platform (near the spot to play ZL to enter domain), you just jump towards the waterfall. The water will sweep you into the rupees. You just need to adjust slightly left or right to get each one.

Reply:
I will see if I can track down the person who originally implemented the Jabu jump dive trick and get their take on it.

Reply:
I spoke with rattus (the author of the Jabu jump dive trick) and he generally agrees with what I said previously. That getting all 4 rupees from the waterfall (or getting any specific rupee, put another way) is actually not so easy, and definitely harder than the Jabu jump dive. He still thinks the jump dive trick in Jabu is worth keeping, and for consistency the ZR jump dive should be a trick as well.

Reply:
Could these two jump dives be the same trick maybe?

Reply:
There are a few other jump dive tricks as well that can't really be combined all together. It'd be super weird then for just jabu and zr to be combined.